http-core/src/main/scala/org/apache/pekko/http/impl/model/parser/CacheControlHeader.scala

```diff
@@ -5,11 +5,10 @@
 package org.apache.pekko.http.impl.model.parser
 
 import org.apache.pekko
-import pekko.parboiled2.Parser
 import pekko.http.scaladsl.model.headers._
 import CacheDirectives._
 
-private[parser] trait CacheControlHeader { this: Parser with CommonRules with CommonActions with StringBuilding =>
+private[parser] trait CacheControlHeader { this: HeaderParser =>
```

Comment:
Somehow the Scala compiler got confused here about the more restricted self-type, not sure what's the issue, but that works for now.